testing/test_BumpHunter.py

```diff
@@ -2,79 +2,107 @@
 # Here, all the values required to pass te test has been validated using the
 # orriginal c++ version of BumpHunter used by ATLAS collaboration.
 
-import pyBumpHunter as BH
-import pytest
-import uproot as upr
+import os
+
 import numpy as np
+import pytest
+
+import pyBumpHunter as BH
 
 # Import the data
-File = upr.open('data/data.root')
-bkg = File['bkg'].arrays(library='np')['bkg']
-data = File['data'].arrays(library='np')['data']
-sig = File['sig'].arrays(library='np')['sig']
-
-# Create the BumpHunter instance
-BHtest = BH.BumpHunter(rang=[0,20],
-                       width_min=2,
-                       width_max=6,
-                       width_step=1,
-                       scan_step=1,
-                       Npe=10000,
-                       Nworker=1,
-                       seed=666)
+
+here = os.path.dirname(os.path.realpath(__file__))
+
+
+def make_datasets(seed):
```

Comment:
Why not like this? Or if a file is used, produce it this way, not with ROOT and C++

Reply:
Please use pathlib.Path instead of manual string manipulation of paths, it's much cleaner and more readable.

Reply:
Good point, I fully agree. _if_ it is needed to access the file actually